lib/rucio/db/sqla/models.py

```diff
@@ -665,6 +665,12 @@ class RSE(BASE, SoftModelBase):
                    CheckConstraint('RSE IS NOT NULL', name='RSES_RSE__NN'),
                    CheckConstraint('RSE_TYPE IS NOT NULL', name='RSES_TYPE_NN'),)
 
+    def restore(self, session=None):
```

Comment:
Sorry for the late comment @arisfkiaras 
Please move that one to the `restore_rse` method directly. (so just to change `.deleted` and `.deleted_at` and then save)
I don't want to put (more) methods into the models directly.